src/iwant_bot/start.py

```diff
@@ -25,27 +24,29 @@ def format_message(message):
     text += ['so you can do http://localhost:8080/?name=me&msg=message']
     text += ['We got these entries so far:\n']
     text += ['Or you can test POST request by command:']
-    text += ['> curl -X POST  --data "user_name=0&channel_id=1&channel_name=2&service_id=3&']
-    text += ['  team_domain=4&team_id=5&text=6&timestamp=7&token=9&user_id=9" http://localhost:8080']
-             #curl -X POST some_url_like_https://kiwislackbot1.localtunnel.me/\n']
+    text += ['> curl -X POST  --data "user_name=0&channel_name=2']
```

Comment:
ok, so you can decide

1. squash the commits yourself to multiple changes, so we don't have to do it on github

2. out of commit for this MR